colcon_test_result/verb/test_result.py

```diff
@@ -82,6 +92,9 @@ def main(self, *, context):  # noqa: D102
                     _output_messages('stdout output', testcase.system_outs)
                     _output_messages('stderr output', testcase.system_errs)
 
+        if context.args.result_files_only:
+            return 0
```

Comment:
When always returning `0` when the new option is used the caller doesn't have a way of outputting only the paths but still get the rc. On the other hand if the rc is still returned the caller can simply decide to ignore it with `|| true`. Therefore I would be in favor of keeping the rc the same as without the option.